### PR TITLE
{lang}[intel/2015b] numba 0.22.1 w/ Python 2.7.11 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Currently, the `import numba` fails with:

```
*** glibc detected *** python: free(): invalid pointer: 0x00002aaac7632660 ***
```

Known issue, something to do with inconsistent static linking, cfr. https://github.com/numba/llvmlite/issues/93

**Comment by boegel** [comment]
added patch fixes the problem, hooray \o/

**Comment by boegel** [comment]
@wpoely86: please review

File: easybuild/easyconfigs/l/LLVM/LLVM-3.6.2-intel-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '3.6.2'
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.6.2-intel-2015b.eb]: why not 3.7?

Comment (boegel) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.6.2-intel-2015b.eb]: numba doesn't work yet on top of LLVM 3.7

**Comment by wpoely86** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1ad1b55f3eddba9c6eea for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/bccad2d9aad9616dbb95 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**